### PR TITLE
Added sass-lint and some basic rules 7.x

### PR DESCRIPTION
**Pull request by carwin**
This is the 7.x version of #82 for issue #56

### CODE REVIEW

**Comment by carwin** [comment]
@wesruv look here too.

**Comment by wesruv** [comment]
supervisor

File: Gruntfile.js [hunk 1]
```diff
@@ -113,8 +120,9 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-contrib-watch');
   grunt.loadNpmTasks('grunt-wiredep');
   grunt.loadNpmTasks('grunt-autoprefixer');
+  grunt.loadNpmTasks('grunt-sass-lint');
   grunt.loadNpmTasks('grunt-eslint');
 
-  grunt.registerTask('default', ['clean','wiredep_create_bower', 'wiredep', 'sass_globbing', 'sass', 'autoprefixer', 'eslint']);
+  grunt.registerTask('default', ['clean','wiredep_create_bower', 'wiredep', 'sass_globbing', 'sass', 'autoprefixer', 'sasslint', 'eslint']);
```
Comment (wesruv) [inline Gruntfile.js]: Methinks it should go before the sass is built. It should get a chance to throw an error before we send it to the compiler.

**Comment by wesruv** [comment]
Should go here too: https://github.com/Lullabot/windup/pull/83/files#diff-35b4a816e0441e6a375cd925af50752cR30

File: Gruntfile.js (hunk 1 above)
Comment (carwin) [inline Gruntfile.js]: That's a good point, I should probably change that for the 8.x branch too if you haven't already.

Comment (wesruv) [inline Gruntfile.js]: It's that way

**Pull request merged**